Meadowlark-js/tests/e2e/helpers/Resources.ts

```diff
@@ -21,7 +21,7 @@ export async function createResource({
     .send(body)
     .then((response) => {
       if (response.body) {
-        console.error(response.body);
+        console.error(`Error on ${endpoint}:\n${response.body}`);
```

Comment:
Should we have an `expect(response.body).toBeUndefined()` or something like that here, so that the test fails in this situation?

Reply:
I like this idea